src/common/components/Tooltip/Skin/index.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import React from 'react';
+import Icon from 'common/components/Icon';
+import Background from '../Background';
+import styles from './styles.less';
+
+type Props = {
+  data: Object,
+};
+
+const SkinTooltip = ({ data: { skin } }: Props) => {
+  return (
+    <Background>
+      <div className={styles.itemHeader}>
+        <Icon size="mini" src={skin.icon} className={styles.tooltipIcon} />
+        <span className={styles.skinNameText}>
+          {skin.name}
+        </span>
+      </div>
+      <div>
+        <span className={styles.skinTypeText}>
```

Comment:
can we get rid of the span and just utlise the parent div for the classname?

Reply:
Yep, let me check it out.